web/src/app/dialogs/FormDialog.js

```diff
@@ -64,6 +64,9 @@ export default class FormDialog extends React.PureComponent {
     onClose: p.func,
     onSubmit: p.func,
 
+    // disables form submit button
+    submitDisabled: p.bool,
```

Comment:
The "Too many messages" error message is actually from the UI that we show if it errors out when you manually hit send since the backend error messages look bad. I think it would be better to remove that though, and make the backend error message better